packages/react/src/header/Header.tsx

```diff
@@ -5,49 +5,68 @@ import { HeaderBrand } from './HeaderBrand';
 import { HeaderContainer } from './HeaderContainer';
 
 export type HeaderProps = {
-	/** The logo to display. */
-	logo?: JSX.Element;
-	/** The heading should be set to the website or service title. */
-	heading: string;
-	/** Used to provide additional information to describe your website or service. */
-	subline?: string;
+	/** The background color of the Header. */
+	background?: 'body' | 'bodyAlt';
 	/** Used to indicate if an application is in a prerelease state. */
 	badgeLabel?: string;
+	/** When using two logos, position the horizontal dividing line 'between' the logos or 'after' them. */
+	dividerPosition?: 'after' | 'between';
```

Comment:
I wonder if this should include logos (2 logos) in the name, since having it non-functioning on the normal, single logo header feels a bit weird

Reply:
We can do something like `secondLogoDividerPosition`? I don't think it'd add much because you have to either look at the docs or leverage TypeScript to learn the values and that's right where the JSDoc explanation is. For me, this falls into the same boat as `fixedHeight` on the Popover where you've got a succinct prop name with extended detail in the JSDoc.